README.md

```diff
@@ -120,15 +121,17 @@ Once you have your user's OAuth token, you can re-use the `OAuth Consumer` and c
 qb_access_token = quickbooks_credentials.access_token
 qb_refresh_token = quickbooks_credentials.refresh_token
 
-access_token = OAuth2::AccessToken.new(::QB_OAUTH2_CONSUMER, qb_access_token, { refresh_token: qb_refresh_token })
+access_token = OAuth2::AccessToken.new(::QB_OAUTH2_CONSUMER, qb_access_token, { :refresh_token => qb_refresh_token })
 ```
 
 ### Access Token Validity and Token Refresh
 
 Each access token is only valid for one hour. The access token and refresh token can be refreshed directly by using OAuth Client:
 
 ```ruby
-new_access_token = access_token.refresh!
+redirect_uri = oauth_callback_quickbooks_url
+
+new_access_token = access_token.refresh!(:redirect_uri => redirect_uri)
```

Comment:
I can do `access_token.refresh!` just fine without providing the `redirect_uri` param.

Reply:
I would have sworn that `redirect_uri` was mandatory in the refresh request, too. I might have dreamt about that... 😂 I fixed it! Thanks